src/es5processor.ts

```diff
@@ -209,10 +217,11 @@ class ES5Processor extends Rewriter {
   emitGoogRequire(varName: string|null, tsImport: string): string {
     let modName: string;
     let isNamespaceImport = false;
-    if (tsImport.match(/^goog:/)) {
+    const nsImport = extractGoogNamespaceImport(tsImport);
+    if (nsImport !== null) {
       // This is a namespace import, of the form "goog:foo.bar".
       // Fix it to just "foo.bar", and save the variable name.
```

Comment:
It looks like this doesn't actually save the variable name, but maybe for some other CL